overview/what-is-post-bid.md

```diff
@@ -2,80 +2,93 @@
 layout: page_v2
 title: Intro to Post-Bid & Get Started
 head_title: What is post-bid and when to use it vs. header bidding.
-
 description: What is post-bid and when to use it vs. header bidding.
-
-pid: 60
-
-top_nav_section: overview
-nav_section: postbid
 sidebarType: 1
-
 ---
 
+# What is Post-Bid?
 
+> Post-bid allows a publisher’s demand sources to compete in one auction based on price after the ad server has declined to choose a direct-sold or exchange-based line item.
 
-# What is post-bid?
-
+`Post-bid` is the configuration scenario where a publisher loads and runs Prebid.js
+in a winning line item's creative. In a post-bid setup, competition among 
+demand sources compete _after_ the ad server has chosen the post-bid line item. Compare this to:
 
-> Post-bid allows a publisher’s mediated demand sources all compete in one auction based on price, after the ad server has picked the post-bid line item. 
+- header bidding, where demand sources compete _before_ the ad server has seen
+the impression.
+- a mediation solution, where post-bid demand sources don't run in a daisy chain;
+they all compete in one single line item based on price.
 
-In post-bid, the competition among your mediated demand sources compete AFTER your ad server has chosen the winning line item (vs. in header bidding, demand sources compete BEFORE your ad server has seen the impression). In post-bid, your mediated demand sources no longer run daisy chain; they all compete in one single line item based on price.
+This diagram summarizes the post-bid scenario:
 
 ![Add Creative to Line Item]({{ site.github.url }}/assets/images/blog/postbid-diagram.png){: .pb-lg-img :}
 
 Steps:
 
 1. The webpage sends an impression to the ad server.
-2. The ad server chooses a winning line item among class 1, exchanges, and the post-bid line items. In this case, the post-bid line item wins because the eCPM on the line item is high (based on historical price) and the higher priority class 1 line items have all exhausted their spend.
+2. The ad server chooses a winning line item among 'class 1' (direct-sold ads), exchanges, and the post-bid line items. In this case, the post-bid line item wins because the eCPM on the line item is high (based on historical price) and the higher priority line items have all exhausted their spend.
```

Comment:
I'm finding this confusing. Are we supposed to know from the diagram that the eCPM is high? Or are we just giving a reason why any post-bid line item would win?

Reply:
I just removed the example. Think lines winning is clear enough.